src/parsers/manifestjson.js

```diff
@@ -623,6 +627,33 @@ export default class ManifestJSONParser extends JSONParser {
     if (this.parsedJSON.homepage_url) {
       this.validateHomePageURL(this.parsedJSON.homepage_url);
     }
+
+    this.validateRestrictedPermissions();
+  }
+
+  validateRestrictedPermissions() {
+    Object.keys(this.restrictedPermissions).forEach((permission) => {
+      const minVersion = this.restrictedPermissions[permission];
+
+      if (
+        (this.parsedJSON.permissions || [])
+          .map((perm) => perm.toLowerCase())
+          .includes(permission)
+      ) {
+        const { firefoxMinVersion: minVersionSetInManifest } =
+          this.getMetadata();
+
+        if (
+          !minVersionSetInManifest ||
+          mozCompare(String(minVersionSetInManifest), minVersion) === -1
```

Comment:
As I describe in a bit more detail in another review comment, I think we may want to consider the scenario where the strict_min_version value set was not valid from a JSONSchema perspective, and in that case don't call mozCompare but also collect the additional RESTRICTED_PERMISSION error.

Reply:
If the strict min version is invalid, an error will be reported (invalid strict min version).

Reply:
I added some test cases. Looks like both errors will be reported.